bootstrap/config/config.go

```diff
@@ -62,6 +63,8 @@ const (
 // UpdatedStream defines the stream type that is notified by ListenForChanges when a configuration update is received.
 type UpdatedStream chan struct{}
 
+var defaultTimeout = 15 * time.Second
```

Comment:
shall we make this configurable?

Reply:
I don't know that we can...unless it is a command line flag or something because we use it to read in the configuration from a uri - so we just need a timeout short enough that we can bootstrap in time.